src/Workleap.OpenApi.MSBuild/SpectralManager.cs

```diff
@@ -86,7 +84,7 @@ private async Task GenerateSpectralReport(string spectralExecutePath, string swa
             await this.AssignExecutePermission(spectralExecutePath, cancellationToken);
         }
 
-        var exitCode = await this._processWrapper.RunProcessAsync(spectralExecutePath, new[] { "lint", swaggerDocumentPath, "--ruleset", rulesetUrl, "--format", "html", "--output.html", htmlReportPath }, cancellationToken);
+        var exitCode = await this._processWrapper.RunProcessAsync(spectralExecutePath, new[] { "lint", swaggerDocumentPath, "--ruleset", rulesetUrl, "--format", "html", "--format", "pretty", "--output.html", htmlReportPath, "--fail-severity=warn" }, cancellationToken);
```

Comment:
It works if you have twice the --format argument?

Reply:
Yup. Oasdiff is weird. It doesn't work if you specify multiple formatter with just one switch